extras/ping/ping.go

```diff
@@ -248,7 +259,11 @@ func (p *Pinger) updateStatistics(pkt *Packet) {
 
 	p.PacketsRecv++
 	if p.RecordRtts {
-		p.rtts = append(p.rtts, pkt.Rtt)
+		p.rtts = append(p.rtts, pkt.Rtt.Seconds())
```

Comment:
The `ping` tool uses milliseconds to represent RTTs, so I think we should do the same to reduce unneeded surprises into users of the tool. What do you think?

Reply:
it's true; I was already doing the conversion on the display. changed it to int64/ms.

Reply:
actually, I think I should keep a float64 with Microseconds / 1e3 to avoid truncating it, wdyt?